Library/PlanRampPricing.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Xml;
+
+namespace Recurly
+{
+    public class PlanRampPricing : RecurlyEntity
+    {
+        /// <summary>3-letter ISO 4217 currency code.</summary>
+        public string Currency { get; set; }
+
+        /// <summary>Represents the price for the Ramp Interval.</summary>
+        public int? UnitAmountInCents { get; set; }
```

Comment:
Yea i thought this may come up.  This is how it was generated for v3.  Im not sure if some decision was made to have them generate like this for safety and then the server does all the validation?  Both unitAmount and StartingBilling cycle were generated by v3 like that.  I dont have an issue removing the nullability tho

Reply:
It _could_ have something to do with like, an integer if its not nullible, and you dont set it, in c# it defaults to 0.  And so if you dont set it, perhaps the thinking was, we would rather allow you to do that and send you a server side error, rather than having it pass in zero... honestly im not sure

Reply:
All of that said, if its nullable, it looks like you dont have to set it, which i dont like...